cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -392,6 +393,7 @@ renderQueryCmd cmd =
     QueryStakeSnapshot' {} -> "query stake-snapshot"
     QueryPoolParams' {} -> "query pool-params"
     QueryKesPeriodInfo {} -> "query kes-period-info"
+    QueryPoolState' {} -> "query pool-state"
```

Comment:
Why are we introducing another query that returns the same thing as the `pool-params` cli command? Surely it would be better to just modify `pool-params` to use your new more efficient query?

Reply:
I experience some confusion because pool params in the ledger refers to something more specific.  What we're actually returning is the pool parameters, future pool parameters and retiring state.

The `query pool-state` command also returns slightly different format allowing for data for multiple stake pools to be returned.

```
{
    "2a20182092016edb6cb818c7e9bde99d03a571a1c724ea6e4c3eb8f3" {
        "futurePoolParams": {..},
        "poolParams": {..},
        "retiring": {..}
    },
    "fc79a939d5986c6565fc5e21bb5dd292261ac5da564c30385b3bb8d4": {
        "futurePoolParams": {..},
        "poolParams": {..},
        "retiring": {..}
    }
}
```

This is the old format:

```
{
    "futurePoolParams": {..},
    "poolParams": {..},
    "retiring": {..}
}
```

Reply:
We can rename `pool-params` to `pool-state` and silently accept `pool-params` (via opt-parse `hidden`). Let's modify the original command we ought not to introduce a new one because it basically is returning the same information.

Reply:
Done!